### ISSUE TITLE
Tasks List

### ISSUE THREAD

**Issue by atherdon** [GroceriStar/calendar#71]
- [ ] our pain with Vadim. We want to update this method, but before someone will start to do it- we should have a good plan that i approve. right now we don't have it. 
https://github.com/GroceriStar/calendar/issues/50
https://github.com/GroceriStar/calendar/pull/48

---
- [ ] Schedule render component have filename Schedule
Easy thing. we just need to rename file and check places, when it was connected and update pathnames there too.

---

- [ ] More details upon request.
> Looks like Sanchit move a few not necessary components, like AddMeal.js
as you can see - that file have inside import of MealPlan form - that we don't have in our project. Also, AddMeal component not called anywhere (I assume).
I want you to clean up our project(*Note*: I didn't tell you delete this component - try to figure out what did i mean)

**Comment by GuiFSs** [comment]
i can do the second and the third task, but for the third one, i'll need more information about what should be in MealForm component

**Comment by atherdon** [comment]
Ok, let's start from second task

**Comment by atherdon** [comment]
Story:
Previosly, most part of this funcitonality was done by Sanchit - while he learned how to code at React.
At some point, we decided to split code, that was made into a few chunks. My aim was to separate meal calendar and work on it as standalone module. it helps us to make it clean, try different approaches, etc.

AddMealForm is a file, that was used at Sanchit project. And we move it only for one reason - without this component - our project will be broken(because our previous project structure was not ideal, but ok).
Actually, at this stage of calendar module - we don't need this form. So main idea was to remove this component. But not just delete it and forget about it. You should be aware and check whole project. because if you just delete this file - it'll break the project - you should prevent it and clean up our code.

tell me if something is not clear @GuiFSs

**Comment by atherdon** [comment]
yes, you're right, but i'm not sure if it not used anywhere

**Comment by GuiFSs** [comment]
ok so, i was testing the "switch mode" button, and i found an error when we get a dish by week, should i open a issue and try to figure it out the problem?  i think it's because the function getRandomWeek in WorkSchedule.js file

**Comment by atherdon** [comment]
yes, please open an issue and explain it here. it'll help us in future

**Comment by atherdon** [comment]
Please follow this link https://github.com/GroceriStar/calendar/invitations

**Comment by GuiFSs** [comment]
> Please follow this link https://github.com/GroceriStar/calendar/invitations
sorry to ask, but what is this for?

**Comment by atherdon** [comment]
@GuiFSs let's discuss here what else we can do at this project right now. Do you have any ideas?

**Comment by GuiFSs** [comment]
ok, but first i would like to know more about the main function of calendar, i mean, correct me if i'm wrong, the objective of calendar is to show the users, suggestions about helthy meals, ingredients, etc, in a certain day and time, so the user can keep it organized and have a better diet

**Comment by GuiFSs** [comment]
well, the only thing i can think now is to have a way of people share this meal plans, i think this is a good way bring more people to the site. But i don't know if this is too complex right now

**Comment by atherdon** [comment]
yeah, i like your idea. will add it to our feature list. and yes - it's not a simple thing

**Comment by atherdon** [comment]
@GuiFSs do you have other tasks?

**Comment by atherdon** [comment]
https://github.com/GroceriStar/react-shopping-list-template/issues/67
https://github.com/GroceriStar/react-shopping-list-template/issues/66
please grab something from here - but notify what you'll take. we are with Vadim working on it too

**Comment by atherdon** [comment]
@GuiFSs tell me if you need help with this task from that repo or if you busy at college - so i'll be able to assign it to someone else
